src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -35,14 +36,19 @@ final class LocoProvider implements ProviderInterface
     private LoggerInterface $logger;
     private string $defaultLocale;
     private string $endpoint;
+    private ?TranslatorBagInterface $translatorBag = null;
 
-    public function __construct(HttpClientInterface $client, LoaderInterface $loader, LoggerInterface $logger, string $defaultLocale, string $endpoint)
+    public function __construct(HttpClientInterface $client, LoaderInterface $loader, LoggerInterface $logger, string $defaultLocale, string $endpoint, TranslatorBagInterface $translatorBag = null)
     {
         $this->client = $client;
         $this->loader = $loader;
         $this->logger = $logger;
         $this->defaultLocale = $defaultLocale;
         $this->endpoint = $endpoint;
+        if (null === $translatorBag) {
+            trigger_deprecation('symfony/loco-translation-provider', '6.1', 'Passing "null" instead of an implementation of "%s" is deprecated, and will be mandatory in Symfony 7.0.', TranslatorBagInterface::class);
```

Comment:
One is more likely to omit the argument than passing it explicitly null
```suggestion
            trigger_deprecation('symfony/loco-translation-provider', '6.1', 'Not passing a "%s" instance as $translatorBag argument to the "%s" constructor is deprecated.', TranslatorBagInterface::class, __CLASS__);
```